### PR TITLE
feat: refactor useEmailRainbow and update Ratio usages

### PR DESCRIPTION
**Pull request by estrattonbailey**
Fixes APP-439

## What changed (plus any additional context for devs)
No reason this needs to be a hook, plus I needed to send different subject lines in each case, and this interface makes that easier.

Hook wasn't used anywhere else, did a grep.

IMG_1306

### CODE REVIEW

File: src/screens/AddCash/providers/Ratio/index.tsx [hunk 1]
```diff
@@ -305,15 +301,23 @@ export function Ratio({ accountAddress }: { accountAddress: string }) {
...
           `Ratio: account recovery clicked`,
           {},
           logger.DebugContext.f2c
         );
-        emailRainbow();
+        emailRainbow({
+          subject: lang.t(
+            lang.l.wallet.add_cash_v2.support_emails.account_recovery
+          ),
+        });
```
Comment (jkadamczyk) [inline src/screens/AddCash/providers/Ratio/index.tsx:320]: That's way better and easier to reuse where you can't use a hook 👍

File: src/utils/emailRainbow.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { debounce } from 'lodash';
+// @ts-expect-error
+import Mailer from 'react-native-mail';
```
Comment (jkadamczyk) [inline src/utils/emailRainbow.ts:3]: It's definitely not in scope of this PR, but this library doesn't seem to be maintained anymore.

**Review by jkadamczyk — APPROVED** [review]
Looks good!

**Review by skylarbarrera — APPROVED** [review]
nice

**Pull request merged**